community/foundation-delegation-program.md

```diff
@@ -83,6 +83,7 @@ compliance screen
 * Dedicated email address so that the Foundation can reach you in the event
 of emergency upgrades and fixes
 * Maintain a fully archival (non pruned) bridge node for both Mainnet Beta and Mocha if selected for the program
+* Not running your infra in Hetzner or OVH 
```

Comment:
should this acronym be clarified @mindstyle85 ?

Reply:
you mean "infra"?

Reply:
fixed it :)

Reply:
i meant OVH but all g